test/e2e/mixed_placement_test.go

```diff
@@ -0,0 +1,350 @@
+							ObservedResourceIndex: "0",
+							FailedPlacements:      failedPlacements,
+							Conditions:            perClusterApplyFailedConditions(rp.Generation),
+						},
+						{
+							ClusterName:           memberCluster3WestProdName,
+							ObservedResourceIndex: "0",
+							FailedPlacements:      failedPlacements,
+							Conditions:            perClusterApplyFailedConditions(rp.Generation),
+						},
+					},
+					SelectedResources:     appConfigMapIdentifiers(),
+					ObservedResourceIndex: "0",
+				}
+				if diff := cmp.Diff(rp.Status, wantStatus, placementStatusCmpOptions...); diff != "" {
+					return fmt.Errorf("RP status diff (-got, +want): %s", diff)
+				}
+				return nil
+			}
+			Eventually(rpStatusUpdatedActual, eventuallyDuration, eventuallyInterval).Should(Succeed(), "Failed to update RP %s status as expected", rpName)
+		})
+
+		// It cannot use ensureRPAndRelatedResourcesDeleted since the configMap is not applied by RP.
+		It("can delete the RP", func() {
+			// Delete the RP.
+			rp := &placementv1beta1.ResourcePlacement{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      rpName,
+					Namespace: appNamespace().Name,
+				},
+			}
+			Expect(hubClient.Delete(ctx, rp)).To(Succeed(), "Failed to delete RP %s/%s", rp.Namespace, rpName)
+		})
+
+		It("should remove controller finalizers from RP", func() {
+			finalizerRemovedActual := allFinalizersExceptForCustomDeletionBlockerRemovedFromPlacementActual(types.NamespacedName{Name: rpName, Namespace: appNamespace().Name})
+			Eventually(finalizerRemovedActual, eventuallyDuration, eventuallyInterval).Should(Succeed(), "Failed to remove controller finalizers from RP %s/%s", appNamespace().Name, rpName)
+		})
+
+		It("remove the RP finalizers", func() {
+			cleanupPlacement(types.NamespacedName{Name: rpName, Namespace: workNamespaceName})
+		})
+	})
+
+	Context("CRP cannot co-own the resources with RP", Ordered, func() {
+		BeforeAll(func() {
+			By("creating work resources")
+			createWorkResources()
+
+			// Create the resources on the member clusters.
+			for idx := range allMemberClusters {
+				memberCluster := allMemberClusters[idx]
+				By(fmt.Sprintf("creating namespace on member cluster %s", memberCluster.ClusterName))
+				ns := appNamespace()
+				Expect(memberCluster.KubeClient.Create(ctx, &ns)).To(Succeed())
+			}
+
+			By("creating an RP")
+			createRP(workNamespaceName, rpName)
+
+			By("should update RP status as expected")
+			rpStatusUpdatedActual := rpStatusUpdatedActual(appConfigMapIdentifiers(), allMemberClusterNames, nil, "0")
+			Eventually(rpStatusUpdatedActual, eventuallyDuration, eventuallyInterval).Should(Succeed(), "Failed to update RP %s status as expected", rpName)
+
+			By("creating a CRP that uses PickAll strategy")
+			createCRP(crpName)
+		})
+
+		AfterAll(func() {
+			ensureRPAndRelatedResourcesDeleted(types.NamespacedName{Name: rpName, Namespace: workNamespaceName}, allMemberClusters)
+			for idx := range allMemberClusters {
```

Comment:
Why do we need to manually clean up the member clusters? After the RP is deleted, CRP can take over?